README.md

```diff
@@ -49,16 +49,22 @@ Finally, to run the project locally:
 npm run dev
 ```
 ### OpenApi documentation
-Open Api documentation is managed through [redoc](https://redocly.com/docs/redoc/). Changes to the documentation are managed through edits to the `openapi/openapi.yaml` file. The `openapi/index.html` is used to serve the rendered documentation. It is generated through the redocly cli and should not be manually edited.
-
-The api serves the documentation on its root path.
-
-- To serve a development version of the rendered documentation while making edits, use `npm run redoc:dev`.
+Open Api documentation is managed through [redoc](https://redocly.com/docs/redoc/). 
+Changes to the documentation are managed through edits to the `openapi/openapi.yaml` file. 
+The `openapi/index.html` is used to serve the rendered documentation. It is generated through the redocly cli and should not be manually edited.
+*It was historically a source-controlled file. However, we found it is more practicle to generate it as part of the API development and production build steps.*
+
+- Running the API development server with `npm run dev` will also rebuild the `openapi/index.html`
```

Comment:
Should we break out this part for 
Local development
- dynamic changes to openapi.yaml (`npm run redoc:dev` on port 8080)
- changes to API implementation (`npm run start:debug` or `npm run dev`)

Deployment
- staging and production (`npm run dev` and `npm run build` on port 3000)

In case it's helpful to have what each npm command does rather than having to mention regeneration of the html file and other info multiple times